.github/workflows/terraform-static-analysis.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       with:
         fetch-depth: 0
     - name: Run Analysis
-      uses: ministryofjustice/github-actions/terraform-static-analysis@433c75e44be4eabb0a6ca573951090b9da4901cf # v15.1.0
+      uses: ministryofjustice/github-actions/terraform-static-analysis@5581a7d1d42cd24fc0308cdab75be22f2d8c9d11 # v15.1.0
```

Comment:
Hi @mikereiddigital if this is v15.2.0 you ought to update the comment to say that too?

Reply:
Thanks @richgreen-moj - I thought those were updated automatically but I'll amend them :)